pkg/controller/infra/infra_controller.go

```diff
@@ -233,8 +214,20 @@ func (c *InfraController) syncCluster(key string) error {
 	// TODO: Deep-copy only when needed.
 	cluster = cluster.DeepCopy()
 
-	c.logger.WithField("cluster", cluster.Name).Infoln("provisioning cluster")
-	// TODO
+	c.logger.WithField("cluster", cluster.Name).Infoln("provisioning cluster infrastructure")
+	ansibleRunner := ansible.NewAnsibleRunner(c.kubeClient)
+	inventoryGenerator := ansible.NewVarsGenerator(cluster)
```

Comment:
We need some name consistency here. The type and the function use "vars generator". The variable uses "inventory generator". The type actually does both vars and inventory generation.

Reply:
inventory generator is a relic of where I originally thought this was going, will adjust.